packages/forms/frontend/sirius-components-forms/src/groups/Group.tsx

```diff
@@ -11,20 +11,25 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 import { ServerContext } from '@eclipse-sirius/sirius-components-core';
-import { makeStyles } from '@material-ui/core/styles';
+import { makeStyles, Theme } from '@material-ui/core/styles';
 import Typography from '@material-ui/core/Typography';
 import ToggleButton from '@material-ui/lab/ToggleButton';
 import ToggleButtonGroup from '@material-ui/lab/ToggleButtonGroup';
 import { useContext, useEffect, useState } from 'react';
 import { PropertySection } from '../propertysections/PropertySection';
 import { ToolbarAction } from '../toolbaraction/ToolbarAction';
-import { GroupProps } from './Group.types';
+import { GroupProps, GroupStyleProps } from './Group.types';
 
-const useGroupStyles = makeStyles((theme) => ({
+const useGroupStyles = makeStyles<Theme, GroupStyleProps>((theme) => ({
   group: {
     display: 'flex',
     flexDirection: 'column',
-    paddingTop: theme.spacing(1),
+    margin: theme.spacing(1),
+    padding: theme.spacing(0.5),
+    borderWidth: ({ borderSize }) => borderSize || 0,
+    borderColor: ({ borderColor }) => borderColor || 'black',
```

Comment:
So you change the default behavior to no visible border to black border, even if the specifier set `None` to the border color? Unless I'm missing something here, when no border color is specified, `transparent` should be used.